apps/backend/api/graphql/mutations/convertGraphqlData.js

```diff
@@ -1,6 +1,7 @@
 import { transform, snakeCase } from 'lodash'
 
 export default function convertGraphqlData (data) {
+  console.log('data', data)
```

Comment:
I know this is a draft but just tagging this one so it will still come up in the PR history if its still around for the actual review